docs/static_site/src/_includes/get_started/linux/clojure/gpu.md

```diff
@@ -1,7 +1,4 @@
-You can use the Maven packages defined in the following dependency to include MXNet in your Clojure
-project. To maximize leverage, the Clojure package has been built on the existing Scala package. Please
-refer to the [MXNet-Scala setup guide]({{'/get_started/scala_setup'|relative_url}}) for a detailed set of instructions
-to help you with the setup process that is required to use the Clojure dependency.
+Please refer to the [MXNet-Clojure setup guide](https://github.com/apache/incubator-mxnet/tree/master/contrib/clojure-package) for a detailed set of instructions to help you with the setup process that is required to use the Clojure dependency.
```

Comment:
Just curious is there any difference between this gpu.md and cpu.md? If not, could we just merge them into one doc?

Reply:
The only difference is the dependency tag, one for cpu and the other for gpu. It makes them not easy to merge.